src/Mod/Path/PathScripts/post/KineticNC_post.py

```diff
@@ -0,0 +1,365 @@
+# ***************************************************************************
+# *   (c) sliptonic (shopinthewoods@gmail.com) 2014                        *
```

Comment:
Should be:  
`# *  Copyright (c) 2014 sliptonic <shopinthewoods@gmail.com>                         *`

Reply:
Do I have to make a new pull request with these changes? Sorry if that is a stupid question, this is my first pull request ever

Reply:
No unnecessary, in the same branch that you've submitted, made the appropriate edit. Then type:
```
git add .
git commit -m "Added requested revision"
git push
```

But you've made the mistake using the master branch to submit your changes and not making a new branch. You probably don't have a Github token setup either to  `git push` from the command line. So whomever the dev that merges this should make the changes for you.

Reply:
You can also make the changes locally and amend the existing commit.  Then do a git push --force 
Github still detects the change and will update the PR

Reply:
At this point it seems the easiest to create a proper pull request from the command line and not via file upload. I will close this request.